RELEASES.md

```diff
@@ -46,11 +46,13 @@ schedule, initially aiming for the bi-weekly releases.
 Release managers of stable releases are responsible for approving and merging backports, tagging
 stable releases and sending announcements about them. This role is rotating on a quarterly basis.
 
-| Quarter |       Release manager        |
-|:-------:|:----------------------------:|
-| 2020 Q1 | Piotr Sikora ([PiotrSikora]) |
-| 2020 Q2 | Piotr Sikora ([PiotrSikora]) |
-| 2020 Q3 | Yuchen Dai ([lambdai])       |
+| Quarter |       Release manager            |
+|:-------:|:--------------------------------:|
+| 2020 Q1 | Piotr Sikora ([PiotrSikora])     |
+| 2020 Q2 | Piotr Sikora ([PiotrSikora])     |
+| 2020 Q3 | Yuchen Dai ([lambdai])           |
+| 2020 Q4 | Christoph Pakulski ([cpakulski]) |
+| 2021 Q1 | Rei Shimizu (@Shikugawa)         |
 
```

Comment:
Those links don't point anywhere right now (see the last line in this file). I think it might be better to inline links here, otherwise we keep missing the link targets.

Reply:
yeah, good call, I'd missed those entirely

Reply:
Nit: `s/shikugawa/Shikugawa/`